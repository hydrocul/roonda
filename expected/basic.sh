$ROONDA_SELF_PATH $ROONDA_TMP_PATH/perl.pl
$ROONDA_SELF_PATH $ROONDA_TMP_PATH/ruby.rb
$ROONDA_SELF_PATH $ROONDA_TMP_PATH/python2.py

#################################################
# perl.pl:
#################################################
# 
# (perl v1)
# 
# (apply print "Hello, world!\n")
# 
# (print (+ 1 2))
# 

#################################################
# ruby.rb:
#################################################
# 
# (ruby v1)
# 
# (apply print "Hello, world!\n")
# 
# (print (+ 1 2))
# 

#################################################
# python2.py:
#################################################
# 
# (python2 v1)
# 
# (apply print "Hello, world!")
# 
# (print (+ 1 2))
# 
