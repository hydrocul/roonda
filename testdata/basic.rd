(sh v1)

(roonda perl.pl)
(roonda ruby.rb)
(roonda python2.py)

<< perl.pl

(perl v1)

(apply print "Hello, world!\n")

(print (+ 1 2))

perl.pl >>

<< ruby.rb

(ruby v1)

(apply print "Hello, world!\n")

(print (+ 1 2))

ruby.rb >>

<< python2.py

(python2 v1)

(apply print "Hello, world!")

(print (+ 1 2))

python2.py >>

