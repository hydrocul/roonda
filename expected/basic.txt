Hello, world!
3Hello, world!
3Hello, world!
3